cmd/build/lint.go

```diff
@@ -31,5 +31,5 @@ func (Lint) glciFix() error {
 }
 
 func (Lint) glciCheck() error {
-	return shr.Run("golangci-lint", "run", "--deadline=15m", "./...", "./apis/...", "./pkg/...")
+	return shr.Run("golangci-lint", "run", "--fix", "--deadline=15m", "./...", "./apis/...", "./pkg/...")
```

Comment:
```suggestion
	return shr.Run("golangci-lint", "run", "--deadline=15m", "./...", "./apis/...", "./pkg/...")
```
Nope, the check fn is supposed to check only. If you want to get rid of it lets discuss it, but that change would just create a duplicate method :)

Reply:
Sorry my bad. I restored the old behavior now.